sdk/src/main/scala/io/horizen/account/AccountSidechainApp.scala

```diff
@@ -148,6 +146,8 @@ class AccountSidechainApp @Inject()
   // Init Sync Status actor
   val syncStatusActorRef: ActorRef = SyncStatusActorRef("SyncStatus", sidechainSettings, nodeViewHolderRef, params, timeProvider)
 
+  override lazy val applicationApiRoutes: Seq[AccountApplicationApiRoute] = customApiGroups.asScala.map(apiRoute => AccountApplicationApiRoute(settings.restApi, apiRoute, nodeViewHolderRef))
```

Comment:
```suggestion
  override lazy val applicationApiRoutes: Seq[ApiRoute] = customApiGroups.asScala.map(apiRoute => AccountApplicationApiRoute(settings.restApi, apiRoute, nodeViewHolderRef))
```
I assume that Seq type must remain the same as in the base class.

Reply:
Right, done